### PR TITLE
DOC-5368: Added the preinstall command to Corda CLI commands

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

**Review by jacob-scott3141 — APPROVED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by yarivus — DISMISSED** [review]
LGTM

File: content/en/platform/corda/5.1/reference/corda-cli/preinstall.md [hunk 1]
```diff
@@ -91,21 +91,21 @@ table th:nth-of-type(2) {
 | Argument | Description                                                                             |
 | --------------------------------------- | ---------------------------------------------------------|
 | \<path\> | The YAML file containing the Kafka, SASL, and TLS configurations. |
-| \-n, \-\-namespace=<namespace> | The namespace in which to look for the Kafka secrets if TLS or SASL is enabled. |
-| \-t, \-\-timeout=<timeout> | The timeout in milliseconds for testing the Kafka connection - defaults to 3000. |
+| \-n, \-\-namespace=<namespace\> | The namespace in which to look for the Kafka secrets if TLS or SASL is enabled. |
```
Comment (yarivus) [inline content/en/platform/corda/5.1/reference/corda-cli/preinstall.md:94]: don't you need an escape character for `<` too ?

**Review by yarivus — COMMENTED** [review]
1 comment

File: content/en/platform/corda/5.1/reference/corda-cli/preinstall.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/5.1/reference/corda-cli/preinstall.md:94]: @yarivus no, it works, please see the preview: https://anna.preview.docs.r3.com/en/platform/corda/5.1/reference/corda-cli/preinstall.html#preinstall

Comment (yarivus) [reply content/en/platform/corda/5.1/reference/corda-cli/preinstall.md:94]: OK thanks 👍

**Review by yarivus — APPROVED** [review]
LGTM

**Review by Niamh25 — APPROVED** [review]

**Pull request merged**